File: Topic Based Problems/01 Fundamentals/1 Beginner/05 Rectangle Perimeter/C/RectanglePerimeter.c
#include <stdio.h>

int main() {
    int length, width;

    printf("Enter length: ");
    scanf("%d", &length);

    printf("Enter width: ");
    scanf("%d", &width);

    printf("Perimeter of rectangle is: %d\n", 2 * (length + width));

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/05 Rectangle Perimeter/Java/RectanglePerimeter.java
import java.util.Scanner;

public class RectanglePerimeter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int length, width;

        System.out.print("Enter length: ");
        length = scanner.nextInt();

        System.out.print("Enter width: ");
        width = scanner.nextInt();

        System.out.println("Perimeter of rectangle is: " + 2 * (length + width));

        scanner.close();
    }
}
